messaging/src/test/java/org/axonframework/common/BuilderUtilsTest.java

```diff
@@ -0,0 +1,125 @@
+package org.axonframework.common;
+
+import org.junit.*;
+
+import java.util.Random;
+
+import static org.axonframework.common.BuilderUtils.*;
+import static org.junit.Assert.*;
+
+public class BuilderUtilsTest {
+
+    private static final int NUMBER_OF_RANDOM_NUMBERS = 256;
+
+    private static void testAssertFails(Runnable r, final String failureMsg) {
+        try {
+            r.run();
+            fail(failureMsg);
+        }
```

Comment:
If the Axon Framework code style is used, you will not have the `catch`, or `else`, on a new line at all.
Would you mind use the code style to format the try-catch and if-else blocks in line with the rest of the code base?

Reply:
Reformatted